sys-libs/glibc/glibc-9999.ebuild

```
@@ -1272,6 +1272,11 @@ glibc_src_test() {
 			ewarn "Skipping extra tests because in systemd-nspawn container"
 			XFAIL_TEST_LIST+=( "${XFAIL_NSPAWN_TEST_LIST[@]}" )
 		fi
+		if [[ "${PORTAGE_NICENESS}" == "39" ]] ; then
```

Rather than testing a Portage-specific variable, you could call `nice` without options.
```suggestion
		if [[ $(nice) == 39 ]] ; then
```

---

Also, max niceness is 19, not 39.

---

Yeah, just realized that, seems I've been unnecessarily using 39 in my `make.conf`, but anything higher than 19 is treated the same as 19.  Fixed and rebased.